src/reporting/market.se

```diff
@@ -211,7 +215,6 @@ def tryFinalizeAutomatedReport():
         return(0)
     self.finalPayoutDistributionHash = self.automatedReporterPayoutDistributionHash
     self.finalizationTime = block.timestamp
-    self.privateRewardBondHolders()
     self.reportingWindow.updateMarketPhase()
```

Comment:
So in tryFinalizeFork(), should I have all of the if statements used in tryFinalizeAutomatedReport(), tryFinalizeLimitedReporting(), and tryFinalizeAllReporting()?